.github/workflows/main.yml

```diff
@@ -21,27 +21,3 @@ jobs:
 
     - name: Run test make target
       run: make test
-
-
-  e2e-pr-blocking:
```

Comment:
Why are we removing this? I think it is good to run quick start test on merge to main. WDYT?

Reply:
As the name suggests `e2e-pr-blocking` should only run before PR is merged. Once all the checks are passed and PR is approved and merged, it will be a waste of resources and time to run those tests again?

Reply:
Yup. Makes sense to me. Okay to remove 👍🏽